spam/spam_tutorial.py

```diff
@@ -128,6 +130,9 @@
 # %% [markdown]
 # ## 2. Write Labeling Functions (LFs)
 
+# %% [markdown]
+# ### A gentle introduction to LFs
```

Comment:
The subsequent sections are all about high level categories of / best practices for LFs, but the user hasn't **seen** an actual LF yet.  I would show one LF first- maybe run it on the dev set, show that it has imperfect accuracy- *then* launch into the below text

Reply:
Also: We have icons / pictures / etc for sprucing these text descriptions up in various docs / spreadsheets; could use them here?